Rakefile

```diff
@@ -116,7 +116,6 @@ namespace :spec do
     :active_support,
     :aws,
     :concurrent_ruby,
-    :cucumber,
```

Comment:
Cucumber and RSpec tests were running twice: once for `spec/ddtrace/contrib/cucumber/\*\*/\*_spec.rb`, which doesn't exist, and another for `spec/datadog/ci/contrib/cucumber/\*\*/\*_spec.rb`, which does.